lib/sauces.test.js

```diff
@@ -0,0 +1,51 @@
+import request from 'supertest';
+import app from './app.js';
+
+it('Returns 404 not found on bad route', async () => {
+  const { text, status } = await request(app).get('/not-found');
+
+  expect(status).toBe(404);
+  expect(text).toBe('Not Found');
+});
+
+it('Returns 404 not found on bad method', async () => {
+  const { text, status } = await request(app)
+  .patch('/api/v1/sauces')
+  .send({ value: 'nonsense' });
+  expect (status).toBe(404);
+  expect(text).toBe('Not Found');
+});
+
+it('Returns a 200 on #GET sauces', async () => {
+  const { text, status } = await request(app)
+  .patch('/api/v1/sauces')
```

Comment:
- `.patch` should be replaced with `.get` for this test
- expect would then change to look for a 200 status